openapi.yml

```diff
@@ -276,6 +276,54 @@ paths:
                 PersonNotFoundError:
                   $ref: "#/components/examples/PersonNotFoundError"
 
+  /v1/persons/{encodedPncId}/alerts:
+    get:
+      tags:
+        - persons
+        - alerts
+      summary: Returns alerts associated with a person.
+      parameters:
+        - $ref: "#/components/parameters/EncodedPncId"
+        - in: query
+          name: page
+          schema:
+            type: number
+            minimum: 0
+            default: 1
+          required: false
+          description: The page number (starting from 1)
+        - in: query
+          name: per_page
+          schema:
+            type: number
+            minimum: 1
+            default: 10
+          required: false
+          description: The maximum number of results for a page
```

Comment:
Thinking these pagination query parameters have now reached my duplication threshold, could we refactor these like `EncodedPncId` in the line above?